tests/boards/nrf/i2c/i2c_slave/testcase.yaml

```diff
@@ -0,0 +1,12 @@
+tests:
+  drivers.i2c_slave.api:
+    depends_on: i2c
+    tags: drivers i2c
+    harness: ztest
+    harness_config:
+      fixture: i2c_loopback
```

Comment:
Yes, it is tied to CI, so we need to know what name is used and provide it when adding DK for execution (together with connected loopbacks).
It can be changed if you insist. `i2c_loopback` was used previously, so setups are configured with that name.

Reply:
Then it's probably not worth the effort. I can live with that.